packages/lib/package.json

```diff
@@ -36,12 +36,14 @@
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-prettier": "^3.1.3",
     "fs": "0.0.1-security",
+    "jest": "^26.6.3",
```

Comment:
I often do this to remove warnings about `ts-jest` being out of date.

Reply:
It's necessary for me to use vs code's test functionality with the project, otherwise it's hard to quickly run individual tests and use the vs code debugger.

If I don't have jest in package.json when I try to run or debug a unit test I get
```
node '/Users/adam/git-repos/mute/jsonld-checker/packages/lib/node_modules/.bin/jest' '/Users/adam/git-repos/mute/jsonld-checker/packages/lib/src/test/jsonld.test.ts' -t 'check should return false if there are dropped terms in a nested json ld'
internal/modules/cjs/loader.js:883
  throw err;
  ^

Error: Cannot find module '/Users/adam/git-repos/mute/jsonld-checker/packages/lib/node_modules/.bin/jest'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:880:15)
    at Function.Module._load (internal/modules/cjs/loader.js:725:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```
Maybe there is a way to integrate tsdx with vs code but I couldn't easily figure it out.

Reply:
Ah got it, sounds very convenient to run tests from VSCode. Is that a plugin or a native feature?

Having two versions of jest creates warning for me locally when running the tests but not a big deal, it's probably fine to leave it

Reply:
Yeah I use plugins firsttris.vscode-jest-runner to run/debug and orta.vscode-jest to continuously run the test while I am working on it and has some other nice features.
<img width="1675" alt="Screen Shot 2021-02-12 at 11 02 47 AM" src="https://user-images.githubusercontent.com/1028600/107798888-565a3480-6d22-11eb-8a2d-cb072738d07b.png">

Reply:
Cool, will check it out !